shell-completion/zsh/_busctl

```diff
@@ -310,5 +310,5 @@ _arguments \
     '--allow-interactive-authorization=[Allow interactive authorization for operation]:boolean:(1 0)' \
     '--timeout=[Maximum time to wait for method call completion and monitoring]:timeout (seconds)' \
     '--augment-creds=[Extend credential data with data read from /proc/$PID]:boolean:(1 0)' \
-    '--num-matches=[Exit after receiving a number of matches while monitoring]:integer' \
+    '--limit-messages=[Exit after receiving a number of matches while monitoring]:integer' \
```

Comment:
Also here.

Reply:
uh, that's a big ask, i have no frickin clue about this language, and it's *weird* with it's `'` use and everything. I cannot reasonably test this, I am pretty sure we should not ask for everyone to learn the syntax of some niche shell to submit a patch like this "zsh" thing.

Reply:
yeah stuff like this can be fixed later, no need to be an RC blocker